### DISCUSSION TITLE
General support for references to the name of a Section/Chapter/etc. rather than its number.

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#1178] [Feature Requests]
In books it is very useful to refer to a section by its name. The Quarto documentation does this, e.g. the link to "Cross references" on this line:

https://github.com/quarto-dev/quarto-web/blob/eed8baacea8448cd2ec60e34250ad2215de6831f/docs/authoring/figures.qmd#L25

But it's done with a hyperlink to an absolute HTML page.

Another example is within a page:

https://github.com/quarto-dev/quarto-web/blob/eed8baacea8448cd2ec60e34250ad2215de6831f/docs/authoring/cross-references.qmd#L26

There's an anchor generated through `## Options {#options}`. It seems to be HTML-specific.

In LaTeX, it's done by `\nameref{sec-blahblah}` and I see that quarto supports this for PDF output. 

Are there plans to make this work with `@sec-blahblah` for other render destinations (e.g. html)? I'm thinking something like `[caption@sec-blahblah]` as a general solution.

**Comment by mcanouil** [comment]
Closing in favour of #1196
